src/main/res/values/styles.xml

```diff
@@ -17,6 +17,5 @@
 
     <style name="WhiteActionButton" parent="Widget.AppCompat.ActionButton.Overflow">
         <item name="android:src">@drawable/ic_more_vert_white_24dp</item>
-        <item name="android:background">?attr/colorPrimary</item>
```

Comment:
Had this in to fix an earlier but, but it makes overflow button background blue even when it should be gray. Also no longer needed.